pkg/cmd/server/origin/master_config.go

```diff
@@ -141,10 +151,25 @@ func BuildMasterConfig(options configapi.MasterConfig) (*MasterConfig, error) {
 	admissionControlPluginNames := []string{"OriginNamespaceLifecycle"}
 	admissionController := admission.NewFromPlugins(privilegedLoopbackKubeClient, admissionControlPluginNames, "")
 
+	var tokenGetter serviceaccount.ServiceAccountTokenGetter
+	if options.KubernetesMasterConfig == nil {
```

Comment:
Why not always create the getter from client and for now use our privileged loopback client.